src/KafkaFlow/Event.cs

```diff
@@ -2,27 +2,23 @@
 {
     using System;
     using System.Collections.Generic;
+    using System.Linq;
     using System.Threading.Tasks;
 
     internal class Event<TArg> : IEvent<TArg>
     {
         private readonly ILogHandler logHandler;
 
-        private readonly IList<Func<TArg, Task>> handlers = new List<Func<TArg, Task>>();
+        private readonly List<Func<TArg, Task>> handlers = new();
 
         public Event(ILogHandler logHandler)
         {
             this.logHandler = logHandler;
         }
 
-        public IEventSubscription Subscribe(Func<TArg, Task> handler)
+        public void Subscribe(Func<TArg, Task> handler)
```

Comment:
Why the `IEventSubscription` was removed?

Reply:
Error in the rebase process. Solved now